### DISCUSSION TITLE
bluetui:  TUI for managing bluetooth devices.

### DISCUSSION THREAD

**Discussion by pythops** [bluez/bluer#138] [Show and tell]
https://github.com/pythops/bluetui

**Comment by potto216** [comment]
Really cool! I'm running it on two RPi4's. Can you direct the controller to advertise from the GUI? I could not figure that out. Also if you have previously paired to another device is it supposed to show up in the scanning window when it is advertising? I think your GUI setup would be great for interacting with a GATT server. Are there plans for that?

**Reply by pythops** [reply]
Thanks.

> Can you direct the controller to advertise from the GUI? 

Do you mean make the adapter `pairable` ?

> if you have previously paired to another device is it supposed to show up in the scanning window when it is advertising

the previously paired devices will be shown in `Paired Devices` section

> I think your GUI setup would be great for interacting with a GATT server. Are there plans for that?

For sure, if someone asks for it, I will implement it.


For further questions, I would suggest to open issues in the [bluetui](https://github.com/pythops/bluetui)  repo

**Reply by potto216** [reply]
Great, I added issues on the [bluetui](https://github.com/pythops/bluetui) repo about it.
